automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataDownloadSpec.scala

```diff
@@ -43,78 +40,65 @@ class DataDownloadSpec extends FreeSpec with ParallelTestExecution with WebBrows
   "Download should reflect visible columns" - {
     "no workspace defaults or user preferences" in {
       val downloadDir = makeTempDownloadDirectory()
-      withWebDriver(downloadDir) { implicit driver =>
```

Comment:
call `withWebDriver` inside the `testMetadataDownload` method

Reply:
good catch here!